File: 后端/src/main/java/com/daocheng/api/product/entity/Product.java
package com.daocheng.api.product.entity;

import com.baomidou.mybatisplus.annotation.*;
import lombok.Data;

import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.List;

@Data
@TableName("product")
public class Product {
    @TableId(type = IdType.AUTO)
    private Long id;
    private Long seriesId;
    private String colorCode;
    private String name;
    private BigDecimal pricePerSqm;
    private Integer stock;
    private String images;  // 存储JSON数组，例如 ["/uploads/1.jpg","/uploads/2.jpg"]
    private String description;
    private Integer sales;  // 销量
    private Integer status; // 0下架 1上架

    @TableField(fill = FieldFill.INSERT)
    private LocalDateTime createdAt;
    @TableField(fill = FieldFill.INSERT_UPDATE)
    private LocalDateTime updatedAt;

    @TableField(exist = false)
    private Series series;
}

File: 前端/src/types/product.ts
export interface Product {
  id: number
  name: string
  brand: '道成' | '飞湃' | '红塬'
  type: '满铺毯' | '方块毯'
  size?: string
  pricePerSqm: number
  images: string
  sales: number
  stock: number
  status: 0 | 1
  description?: string  // 新增（后端可能返回）
  // 移除 createdTime，因为列表接口不返回
}